### PR TITLE
Block entity networking

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
added an amazing block entity networking module. 

0.7.0 is the new version

### CODE REVIEW

File: fabric-networking-blockentity-v0/src/main/java/net/fabricmc/fabric/mixin/networking/blockentity/MixinClientPlayNetworkHandler.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+						return;
+					}
+
+					if (otherIdObj.equals(id)) {
+						serializable.fromClientTag(packet.getCompoundTag());
+					}
+				}
+			}
+
+			info.cancel();
```
Comment (shedaniel) [inline fabric-networking-blockentity-v0/src/main/java/net/fabricmc/fabric/mixin/networking/blockentity/MixinClientPlayNetworkHandler.java:65]: Please error if the id type isn't 127.

Comment (BoogieMonster1O1) [reply fabric-networking-blockentity-v0/src/main/java/net/fabricmc/fabric/mixin/networking/blockentity/MixinClientPlayNetworkHandler.java:65]: necessary?

Comment (shedaniel) [reply fabric-networking-blockentity-v0/src/main/java/net/fabricmc/fabric/mixin/networking/blockentity/MixinClientPlayNetworkHandler.java:65]: yeah

Comment (shedaniel) [reply fabric-networking-blockentity-v0/src/main/java/net/fabricmc/fabric/mixin/networking/blockentity/MixinClientPlayNetworkHandler.java:65]: or just warn

**Comment by BoogieMonster1O1** [comment]
Will need to rewritten as this pull request updating yarn too. Don't have the patience to fix merge conflicts.

**Pull request closed without merging**